<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationView</class>
 <widget class="QWidget" name="ConfigurationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Network File</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="networkFile">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="importNetwork">
              <property name="text">
               <string>Import...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Input File</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputFile">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="chooseInput">
              <property name="text">
               <string>Import...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Batch Input Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="batchFolder">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="chooseBatchFolder">
              <property name="text">
               <string>Open...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>DeepSymbol</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Robustness Type</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="robustnessType">
                  <item>
                   <property name="text">
                    <string>LINFBALL</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>BRIGHTNESS</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>CUSTOMBOX</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1,1">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Perturbation</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="deltaSlider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="deltaSpin"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0">
              <item>
               <widget class="QPushButton" name="run">
                <property name="text">
                 <string>Run</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="batchRun">
                <property name="text">
                 <string>BatchRun</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Eran</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Robustness Type</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>LINFBALL</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,1,1">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Perturbation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="epsilonSlider">
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                  <property name="pageStep">
                   <number>100</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="epsilonSpin">
                  <property name="decimals">
                   <number>5</number>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item alignment="Qt::AlignVCenter">
                 <widget class="QPushButton" name="eranRunButton">
                  <property name="text">
                   <string>Run</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item alignment="Qt::AlignVCenter">
                 <widget class="QPushButton" name="eranBatchRunButton">
                  <property name="text">
                   <string>BatchRun</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>PLANET</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_28">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Robustness Type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_2">
                <item>
                 <property name="text">
                  <string>LINFBALL</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,1,1">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Perturbation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="PLANETDeltaSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="PLANETDeltaSpin"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="1,1">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Target Number:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="PLANETTargetNumberSpin">
                <property name="maximum">
                 <number>9</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,1,1">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Unsmoothness:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="PLANETUnsmoothnessSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="PLANETUnsmoothnessSpin"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QPushButton" name="runPLANETButton">
              <property name="text">
               <string>Run PLANET</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
